shiftcommerce-rails.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "rails", "~> 5.1"
   spec.add_dependency "activemerchant", "~> 1.54"
   spec.add_dependency "meta-tags", "~> 2.4"
+  spec.add_dependency "rack-timeout"
```

Comment:
It might be worth specifying a version here in an unlikely case a future release of `rack-timeout` ships with API changes.